package.json

```diff
@@ -32,12 +32,16 @@
     "!**/.*"
   ],
   "scripts": {
+    "lint:changelog": "auto-changelog validate --prettier",
```

Comment:
What do you think about calling these three scripts automatically in the `lint` script?